### DISCUSSION TITLE
Migrating users from DNN to Oqtane

### DISCUSSION THREAD

**Discussion by bjcam554** [oqtane/oqtane.framework#2509] [Q&A]
Hi
I am considering using Oqtane, but how can the users in a DNN site be migrated to Oqtane? We have other SQL Express databases in the server storing user specific content which use the DNN userid. We would need to continue to use these databases but the users would need to be migrated to Oqtane with the same DNN userid stored somewhere. If we need to create a lookup table for this that's fine, but how to migrate the users complete with logins?
Thanks

**Comment by sbwalker** [comment]
DNN used the original .NET Framework Membership service whereas Oqtane uses the newer .NET Core Identity service. The following article describes the differences:

https://learn.microsoft.com/en-us/aspnet/core/migration/proper-to-2x/membership-to-core-identity?view=aspnetcore-7.0

The article does not include the ability to migrate passwords - users would need to reset their password upon first login. 

If you do some more research you may be able to find a utility which migrates passwords as well, but I was not able to find one myself.

**Reply by bjcam554** [reply]
Thank you
